src/components/Recidiviz/RecidivizSlider/RecidivizSlider.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import RecidivizSlider from './RecidivizSlider';
+
+jest.mock('../../../lib/static/recidivismSlider.js');
+
+it('renders the RecidivizSlider component', () => {
```

Comment:
More test possibilities: dragging to known boundaries in the dataset and ensuring the proper number of person icons are colored red.

Reply:
https://github.com/Recidiviz/recidiviz.com/pull/22#discussion_r186476649